### ISSUE TITLE
Add RetroAchievements support

### ISSUE THREAD

**Issue by nfp0** [libretro/bsnes#27]
All bsnes and higan cores in Retroarch, besides this one, support RetroAchievements (even higan Accuracy and nSide Balanced, contrary to the info in the docs).

It would be great to have this excellent up-to-date core also supporting RetroAchievements. Maybe code from the other bsnes/higan cores can be ported to this one to save time.

**Comment by nfp0** [comment]
According to a maintainer on the upstream repository the missing implementation is

**Comment by nfp0** [comment]
According to a contributor on the upstream repository, the missing implementation is this:
https://github.com/libretro/bsnes/blob/3cc8c589ccdc18cc243fc25253e6b9ad7f085b71/bsnes/target-libretro/libretro.cpp#L711-L721

Byuu says he won't do it himself but he'll be happy to accept a pull request if anyone implements this.
Is this a tall order, considering all previous higan/bsnes cores already implemented this?

**Comment by 1985a** [comment]
I don't know why, but this was working to my few versions back,now it doesn't

**Comment by nfp0** [comment]
@1985a Are you sure you weren't using one of the older bsnes/higan/nSide cores?
This core didn't exist a while back.

**Comment by 1985a** [comment]
I was using it the actual version of bsnes. Then one day it just gone, RA didn't work.

**Comment by nfp0** [comment]
Are you talking about the bsnes 094 version? That's still available. It was renamed to bsnes 2014, and yes, that old version supports RetroAchievements.

**Comment by 1985a** [comment]
no, I'm not using that version. Few versions below, I remember using bsnes, almost when it came the first time, when byuu, said I'm working with some guys from Retroarch to bring bsnes as a core.

They are some logs on Github. Look for Jamiras, he is the guy who has been working on RA.

**Comment by nfp0** [comment]
I've searched and I haven't found anything from Jamiras in this or the upstream repo.
Approximately how long ago was that? Months? Years?
